f1-telemetry-display/src/ui/fmt.rs

```diff
@@ -60,6 +61,14 @@ pub fn set_team_color(team: Team) {
     color_set(TEAM_COLOUR_OFFSET + team.id() as i16);
 }
 
+pub fn set_green() {
+    color_set((STATUS_COLOUR_OFFSET + 1) as i16);
+}
+
+pub fn set_red() {
+    color_set((STATUS_COLOUR_OFFSET + 4) as i16);
+}
```

Comment:
i know its probs bad explicitly setting colours by name but reusing status colours for non status things could have shenanigans